packages/create-sitecore-jss/src/initializers/react-native/index.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import { prompt } from 'inquirer';
+import { prompts, ReactNativeAnswer } from './prompts';
+import { Initializer } from '../../common/Initializer';
+import { transform } from '../../common/steps';
+import { ReactNativeArgs } from '../../common/args/base';
+
+export default class ReactNativeInitializer implements Initializer {
+  get isBase() {
+    return true;
+  }
+
+  async init(args: ReactNativeArgs) {
+    const defaults = args.yes
+      ? {
+          appName: 'sitecore-jss-react-native',
+          hostName: 'https://cm.jss.localhost',
+          language: '',
+        }
+      : {};
+
+    const answers = await prompt<ReactNativeAnswer>(prompts, { ...defaults, ...args });
+
+    const mergedArgs = {
+      ...args,
+      ...answers,
+    };
+
+    const FILTER_REGEXP = /.(jar)$/;
```

Comment:
Same issue as `.pdf`s where it has %> encoded so we want to copy it over, but skip EJS.
It actually needs to go this logic here instead of being skipped completely like it is now: https://github.com/Sitecore/jss/blob/4dff9d16ac57606d7506f0f8396543971e47f716/packages/create-sitecore-jss/src/common/steps/transform.ts#L189

Should we add `jar` to the `ASSET_REGEX`?

Reply:
Sounds like a sensible name/change to me. We have something to make this better (handle both "skip file for copy" and "skip file for ejs processing") in our improvements task.